cmd/kpod/load.go

```diff
@@ -54,7 +55,11 @@ func loadCmd(c *cli.Context) error {
 	}
 
 	args := c.Args()
-	if len(args) > 0 {
+	var image string
+	if len(args) == 1 {
+		image = args[0]
+	}
+	if len(args) > 1 {
```

Comment:
What happens if the user passes no arguments?

Reply:
The user is not required to pass in an argument. This is only for the case where the user wants to load only one image from an oci archive that ha smany.

Reply:
Nit: this should be an else-if because it's exclusive of len(args) == 1

Reply:
@umohnani8 I agree with @mtrmac Change message to say exactly 1

Reply:
> Change message to say exactly 1

That has already happened.